lib/Models/ScheduledTask.php

```diff
@@ -1,7 +1,6 @@
 <?php
 
 namespace MailPoet\Models;
-
```

Comment:
This commit has also many unrelated whitespace coding style changes. Why? They shouldn't be part of this commit, probably not even of this pull request.